src/core/exceptions.py

```diff
@@ -1,14 +1,28 @@
-
 class MeasureSoftGramCoreException(Exception):
     """Base MeasureSoftGram Core exception"""
+
     pass
 
 
 class InvalidMetricValue(MeasureSoftGramCoreException):
     """Raised when a invalid metric value is provided to the interpretation function"""
+
     pass
 
 
 class InvalidInterpretationFunctionArguments(MeasureSoftGramCoreException):
     """Raised when invalid arguments are provided to an interpretation function"""
+
+    pass
+
+
+class InvalidEqualityOfWeightAndValues(MeasureSoftGramCoreException):
```

Comment:
A exception não precisa começar com "Invalid", vocês podem tentar descrever melhor. "Invalid Equality Of Weight And Values" fica um pouco estranho tentem pensar no que realmente significa essa exception. Que tal "ValuesAndWeightsOfDifferentSizes"?